legendapp/client
# Tile switching on client

To test:
1. switch to tile-switching branch
2. bower update openlayers-map
3. try it out in the settings. Make sure you have features on your map to ensure proper z-indexing of layers

Enjoy! We're now 2 features away from being feature-complete here...
